client/src/js/references/components/Detail/Targets/__tests__/Targets.test.js

```diff
@@ -1,14 +1,17 @@
 jest.mock("../../../../../utils/utils");
 
 import { checkRefRight } from "../../../../../utils/utils";
-import { mapStateToProps, mapDispatchToProps, Targets } from "../Targets";
+import { AddTarget } from "../Add";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['AddTarget' is defined but never used.](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=5211749)